packages/fungi/src/MessageEvaluation/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { MessageActions, MessageActionsProps } from "@lg-chat/message-actions";
+
+export type MessageEvalProps = Pick<MessageActionsProps, "onRatingChange">;
+
+export const MessageEvaluation: React.FC<MessageEvalProps> = ({
+  onRatingChange,
+}) => <MessageActions onRatingChange={onRatingChange} />;
```

Comment:
Is there a way to make the rating more obvious? I couldn't tell that I had clicked the button due to the colors being similar:
<img width="306" height="121" alt="Screenshot 2025-09-16 at 4 10 53 PM" src="https://github.com/user-attachments/assets/c8f74542-0b34-449e-86f3-0598435e0320" />